controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1679,6 +1680,53 @@ bool master_thread::handle_cmdu_1905_topology_response(const std::string &src_ma
         }
     }
 
+    // The reported neighbors list might not be correct since the reporting al_mac hasn't received 
+    // a Topology Discovery from its neighbors yet. Therefore, remove a neighbor node only if more
+    // than 65 seconds have passed since we added this node. This promise that the reported al_mac
+    // will get the Topology Discovery messages from its neighbors and add them to the report.
+    auto last_state_change_timestamp = database.get_last_state_change(src_mac);
+    if (last_state_change_timestamp + std::chrono::seconds(65) < std::chrono::steady_clock::now()) {
```

Comment:
Don't use hard coded values - I suggest to move `DISCOVERY_NOTIFICATION_TIMEOUT_SECONDS` to a common place and use this here instead. In addition, you need to say in the comment why specifically 65 seconds.
Its 60 seconds timeout + 5 seconds grace period?

Reply:
yes, it is grace. I did what you asked.